app/coffee/LoggerSerializers.coffee

```diff
@@ -1,4 +1,15 @@
 module.exports =
+	apply: (logger) ->
+		serializers = logger.logger.serializers
+		serializers.docs = @docs
+		serializers.files = @files
+		if logger.isProduction || process.env['TERSE_LOGS']?
```

Comment:
I would be tempted to have consistent logging experience between env's and drop this if, unless you can think of a good reason?

Reply:
How about
`unless process.env['VERBOSE_LOGS']?`

Reply:
In our logger we use 'LOG_LEVEL'  to specify how much to log out. I think `VERBOSE_LOGS` would get a bit lost here in the doc updater. If you think we still need the option of full log lines being printed you could log them out at the debug level https://github.com/trentm/node-bunyan#levels

Reply:
I think `DEBUG` is the default for the dev env? In which case, it's probably best just to drop the condition altogether as I think generally it's better to not have them.

If you're deep enough inside document-updater to need those lines, you're probably also deep enough to comment out the stuff in `LoggerSerializers` instead. WDYT?

Reply:
... have made a change to that effect